reports.py

```diff
@@ -242,6 +261,117 @@ def report_names():
     return _REPORTS.keys()
 
 
+def label_html(label):
+  """Returns html for rendering a Label."""
+  return '<span class="label-%s">%s</span>' % (label.key, label.name)
+
+
+def html_garden():
+    issues = database.get_issues()
+    c_groups = collections.defaultdict(list)
+    predicate = lambda issue: is_open(issue) and not (
+        has_team_label(issue) or has_label(issue, "release"))
+    for issue in filter(predicate, issues):
+        categories = category_labels(issue["labels"])
+        if not categories:
+            categories = ["uncategorized"]
+        for c in categories:
+            c_groups[c].append(issue)
+
+    p = html_writer.HTMLWriter()
+    css = """
```

Comment:
I'm wary that this will scale up to a large mess when we start to add more things to it.

Reply:
Possibly. UI code is always a mess, because you want to control so many aspects. I did not want to build a framework for making it easy to get small CSS blocks until I knew what I needed.

Reply:
Moot now, since we want to think about moving to flask